# Request
float32 degree_turn
---
# Response
sensor_msgs/Image camera_image